File: Selenium/Activities/Activity5.py
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.common.action_chains import ActionChains

with webdriver.Firefox() as driver:
    driver.get("https://v1.training-support.net/selenium/input-events")
    print("the title is :", driver.title)

    actions = ActionChains(driver)
    actions.click().pause(3).perform()

    text1 = driver.find_element(By.CLASS_NAME, "active")
    print(text1.text)

    actions.double_click().pause(2).perform()

    text2 = driver.find_element(By.XPATH, "//div[@class='active']")
    print(text2.text)

    actions.click().pause(2).perform()

    text3 = driver.find_element(By.XPATH, "//div[@class='active']")
    print(text3.text)

    driver.quit()

File: Selenium/Activities/Activity5.java
package activities;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.openqa.selenium.interactions.Actions;

public class Activity5 {

	public static void main(String[] args) {

		WebDriver driver = new FirefoxDriver();
		driver.get("https://v1.training-support.net/selenium/input-events");

		System.out.println("The Title of the page is  : " + driver.getTitle());

		Actions actionClass = new Actions(driver);
		actionClass.click().pause(1000).build().perform();

		String text = driver.findElement(By.xpath("//div[@class='active']")).getText();
		System.out.println(text);

		actionClass.doubleClick().pause(1000).build().perform();

		String text2 = driver.findElement(By.xpath("//div[@class='active']")).getText();
		System.out.println(text2);

		actionClass.click().pause(1000).build().perform();

		String text3 = driver.findElement(By.xpath("//div[@class='active']")).getText();
		System.out.println(text3);

		driver.quit();
	}

}
